tests/test_check_obsfiles_ert_yaml.py

```diff
@@ -0,0 +1,259 @@
+                WCONINJH(
+                    date=value["date"],
+                    well_name=value["WELL_NAME"],
+                    inj_type="GAS",
+                    status=value["WSTAT"],
+                    rate=value["WGIR"],
+                    bhp=value["WBHP"],
+                    thp=value["WTHP"],
+                    inj_control_mode=schedule._inj_control_mode,
+                )
+            )
+
+
+    #vfp_tables = self.get_vfp()
+    vfp_tables = {'K-3H': '1*', 'F-1H': '1*', 'F-2H': '1*'}
+    for _, value in schedule._df_production_data.iterrows():
+        #start_date = self.get_well_start_date(value["WELL_NAME"])
+        start_date = date(2005,10,1)
+        if value["TYPE"] == "OP" and start_date and value["date"] >= start_date:
+            schedule.append(
+                WCONHIST(
+                    date=value["date"],
+                    well_name=value["WELL_NAME"],
+                    status=value["WSTAT"],
+                    prod_control_mode=schedule._prod_control_mode,
+                    vfp_table=vfp_tables[value["WELL_NAME"]],
+                    oil_rate=value["WOPR"],
+                    water_rate=value["WWPR"],
+                    gas_rate=value["WGPR"],
+                    bhp=value["WBHP"],
+                    thp=value["WTHP"],
+                )
+            )
+###############################################################################################                   
+###############################################################################################                                   
+    # TODO We intend to run _create_observation_file but I wasn't able to import the function
+    #      
+    training_set_fraction = 0.75
+    num_dates = len(schedule.get_dates())
+    num_training_dates = round(num_dates * training_set_fraction)
+
+    export_settings = [
+        ["_complete", 1, num_dates],
+        ["_training", 1, num_training_dates],
+        ["_test", num_training_dates + 1, num_dates],
+    ]
+
+
+
+    file_root = pathlib.Path("./tests/observation_files/observations")
+    obs_export_types = ["yamlobs", "ertobs"]
+    print("WRITING the files")
+    for obs_export_type in obs_export_types:
+        for setting in export_settings:
+            export_filename = f"{file_root}{setting[0]}.{obs_export_type}"
+            template = _TEMPLATE_ENVIRONMENT.get_template(
+                f"observations.{obs_export_type}.jinja2"
+            )
+            with open(export_filename, "w") as fh:
+                fh.write(
+                    template.render(
+                        {
+                            "schedule": schedule,
+                            "error_config": config.flownet.data_source.simulation.vectors,
+                            "num_beginning_date": setting[1],
+                            "num_end_date": setting[2],
+                        }
+                    )
+                )
+                        
```

Comment:
This lines are meant to replicate  _create_observation_file. However,   I wasn't able to import  _create_observation_file  from flownet.ert  . It should be possible but I don't know how to do it.

Reply:
Just had chat with you :) Should be solved.